singer/pom.xml

```diff
@@ -200,6 +200,17 @@
 			<artifactId>metrics-core</artifactId>
 			<version>4.0.2</version>
 		</dependency>
+        <dependency>
+            <groupId>com.kohlschutter.junixsocket</groupId>
+            <artifactId>junixsocket-core</artifactId>
+            <version>2.4.0</version>
+            <type>pom</type>
```

Comment:
Why are we using POM dependency?

Reply:
That was what was provided in https://kohlschutter.github.io/junixsocket/dependency.html
It seems that this dependency is a parent pom.